docs/source/configuration/slots.md

```diff
@@ -0,0 +1,379 @@
+});
+```
+
+The following tree structure diagram illustrates the result of the second registration.
+
+```text
+Slot (`name`=`aboveContent`)
+├── SlotComponent
+│   ├── `slot`=`aboveContent`
+│   ├── `name`=`header`
+│   ├── `component`=PageHeader
+│   └── predicate of "only appear under `/de/about`"
+└── SlotComponent
+    ├── `slot`=`aboveContent`
+    ├── `name`=`header`
+    ├── `component`=PageHeader
+    └── predicate of "only appear when the content type is either a Document or News Item"
+```
+
+Finally, let's register another slot component in the same slot, with the same name, but with a different component and without a predicate.
+
+```ts
+config.registerSlotComponent({
+  slot: 'aboveContent',
+  name: 'header',
+  component: 'DefaultHeader',
+});
+```
+
+The following tree structure diagram illustrates the result of the third registration.
+
+```text
+Slot (`name`=`aboveContent`)
+├── SlotComponent
+│   ├── `slot`=`aboveContent`
+│   ├── `name`=`header`
+│   ├── `component`=PageHeader
+│   └── predicate of "only appear under `/de/about`"
+├── SlotComponent
+│   ├── `slot`=`aboveContent`
+│   ├── `name`=`header`
+│   ├── `component`=PageHeader
+│   └── predicate of "only appear when the content type is either a Document or News Item"
+└── SlotComponent
+    ├── `slot`=`aboveContent`
+    ├── `name`=`header`
+    └── `component`=`DefaultHeader`
+```
+
+When the slot components with the same name and component under a given slot have all of their predicates return `true`, then that component will render in the slot.
+Else, if there are slot components with the same name, but with a different component, and without predicates, then that different component will render.
+
+Thus the example slot renderer will have the following behavior.
+
+-   When both a user visits the route beginning with `/de/about`, and the content type is either a Document or News Item, then the component `PageHeader` will render in the `aboveContent` slot.
+-   When one or both of the predicates are false, then the component `DefaultHeader` will render in the `aboveContent` slot.
+
+    ```{tip}
+    In our example, if we had not registered the third slot component—the one without predicates—and when either of the first two slot components' predicates return `false`, then no component would render.
+    ```
+
+```{todo}
+The order in which the slot components render is governed by the order in which they were registered.
+
+You can change the order of the defined slot components for a different slot using the API.
+You can even delete the rendering of a registered slot component using the API.
+
+[@sneridagh, this section needs to be explicit.
+From our conversation yesterday, I think you said that the last slot component registered *per slot* wins.
+Is that correct?
+This has further implications, such as whether this rule applies per slot component's name, component, and predicates. --@stevepiercy]
+```
```

Comment:
This TODO section needs revision, and then I think the docs will be done. If anyone can provide details, I can write it up in narrative form.